images/stylelint/package.json

```
@@ -1,12 +1,12 @@
 {
   "dependencies": {
-    "stylelint": "13.13.1",
+    "stylelint": "14.6.1",
     "stylelint-config-prettier": "9.0.3",
-    "stylelint-config-recommended": "5.0.0",
-    "stylelint-config-recommended-scss": "4.3.0",
-    "stylelint-config-standard": "22.0.0",
-    "stylelint-order": "4.1.0",
+    "stylelint-config-recommended": "7.0.0",
+    "stylelint-config-recommended-scss": "6.0.0",
+    "stylelint-config-standard": "25.0.0",
+    "stylelint-order": "5.0.0",
     "stylelint-prettier": "1.2.0",
```

[suggest] How about updating [stylelint-prettier](https://www.npmjs.com/package/stylelint-prettier) also?

```suggestion
    "stylelint-prettier": "2.0.0",
```

---

That's a good idea. Updating all related packages at once.